src/modules/commentCounter.js

```diff
@@ -0,0 +1,17 @@
+/* const fetchCommentsCount = async (itemId) => {
+  const apiUrl = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/Y9oi34w1aSiFtzn6asAJ/comments?item_id=';
+  const response = await fetch(`${apiUrl}${itemId}`);
+  const comments = await response.json();
+  return comments.length;
+};
+
+export default fetchCommentsCount; */
```

Comment:
Please refrain for leaving comment on your code base it makes your code look dirty.